RadixWallet/Clients/AccountsClient/AccountsClient+Live.swift

```diff
@@ -14,18 +14,20 @@ extension AccountsClient: DependencyKey {
 			}
 		}
 
-		let getAccountsOnCurrentNetwork: GetAccountsOnCurrentNetwork = {
-			try await profileStore.network().accounts
-		}
-
 		let getCurrentNetworkID: GetCurrentNetworkID = { await profileStore.profile.networkID }
 
-		let getAccountsOnNetwork: GetAccountsOnNetwork = { try await profileStore.profile.network(id: $0).accounts }
+		let getAccountsOnNetwork: GetAccountsOnNetwork = {
```

Comment:
I think we ought to rename the "get accounts" methods "getUnhiddenAccounts" to make it clear in code also a reminder that accounts can be hidden.

Reply:
I deliberately wanted to have just Accounts, meaning the accounts that can be used, don't want to put there "unhidden", or any other qualifier, maybe in the future we will have some other conditions for not showing some account.

Reply:
tribal knowledge, so add `/// comment` ?

Reply:
Since we are using "hide" as an alternative to deletion, I think it is quite clear that any unadorned method will just deal with the non-hidden accounts.